src/components/FeedPortal/components/TeamStandupCard.jsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+import './TeamStandupCard.css';
+
+const TeamStandupComponent = ({ userStandup }) => {
+  let standup = userStandup.standup;
+  let healthStatus;
+  switch (standup.progress_sentiment) {
+    case 'red':
+      healthStatus = 'Trouble Ahead!';
+      break;
+    case 'yellow':
+      healthStatus = 'Nervous';
+      break;
+    case 'green':
+      healthStatus = 'Great!';
+      break;
+    default:
+      healthStatus = 'No Response Yet';
+      break;
+  }
+  let standupObjects = [
```

Comment:
Suggestion:
- Remove colons from the text, and place it on line 35
- Downcase values for label, capitalize with styles